plugin.py

```diff
@@ -24,6 +26,20 @@ class LspCssPlugin(NpmClientHandler):
     )
 
     @classmethod
-    def minimum_node_version(cls) -> Tuple[int, int, int]:
-        # this should be aligned with VSCode's Nodejs version
-        return (14, 0, 0)
+    def required_node_version(cls) -> str:
+        return '>=14'
+
+    def on_ready(self, api: ApiWrapperInterface) -> None:
+        session = self.weaksession()
+        if not session:
+            return
+        self.resolve_custom_data_paths(session)
+
+    def resolve_custom_data_paths(self, session: Session) -> None:
+        custom_data_paths = session.config.settings.get('css.customData')  # type: List[str]
+        resolved_custom_data_paths = []  # type: List[str]
+        for folder in session.get_workspace_folders():
+            # Converting to URI as server can't handle reading the content if it's a file path.
+            resolved_custom_data_paths.extend(
+                [filename_to_uri(path.abspath(path.join(folder.path, p))) for p in custom_data_paths])
```

Comment:
same question as for https://github.com/sublimelsp/LSP-html/pull/38#discussion_r1008758990

but this time it is a bit different...
Do you think it would be good to warn users if the path doesn't exist?

Reply:
To be fair I'm not even sure if the warning was a good idea.

If you have multiple folders in the project and using this feature then you are bound to get warnings which you probably don't care about.

Or if you define customData globaly but only have the data files in some projects then you'll also get spammed with warnings.